lib/l10n/L.dart

```diff
@@ -1498,6 +1498,16 @@ class L {
         );
   }
 
+  String get episode_filter_downloaded_label {
```

Comment:
Adding a new label here is good; however, I can't see that you have then gone on and generated the .arb files that contains the translations. Take a look at the [TRANSLATION.md](https://github.com/amugofjava/anytime_podcast_player/blob/master/TRANSLATION.md) file.